src/main/java/yahav/photomosaic/ImagePixels.java

```diff
@@ -19,8 +19,8 @@
     ArrayList<Rectangle2D> squareInput = new ArrayList<>(); //array of input squares to compare to source images
 
     public ImagePixels() throws FileNotFoundException {
-        squarePixelColors = getQuadrantColors();
-        //allImgClrs = getSrcImgAvgColors();
+        //squarePixelColors = getQuadrantColors(); //list of every box's average color of the input image
+        //allImageColors = getSourceImageColors(); //list of average color of every source image
```

Comment:
You've commented this out, which would probably make `closestColorDifference()` crash?